pkg/dockerutil/testdata/docker-compose.override.yml

```diff
@@ -5,7 +5,7 @@ services:
     container_name: ddev-test-foo
     image: DDEV_FOOIMAGE
     volumes:
-      - "./data:/db"
+      - "./import-db:/db"
```

Comment:
This is no longer correct. Needs to be "${DDEV_IMPORTDIR}:/db" right? I wonder how the test works right.